<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>729</width>
    <height>605</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sqlite MVersion</string>
  </property>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>131</width>
     <height>32</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Таблица</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="CreateTable">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>10</y>
     <width>131</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Создать таблицу</string>
   </property>
  </widget>
  <widget class="QPushButton" name="DropTable">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>10</y>
     <width>131</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Удалить Таблицу</string>
   </property>
  </widget>
  <widget class="QLabel" name="Create">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:24pt;&quot;&gt;Создание таблицы&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:24pt;&quot;&gt;&lt;br/&gt;&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="View">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>320</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:24pt;&quot;&gt;Просмотр&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>250</y>
     <width>671</width>
     <height>33</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLineEdit" name="DBpath">
      <property name="text">
       <string>Введите путь до базы данных:/Users/maksimgordienko/Qt1/Sql/RPM13/test</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="ConnectDatabase">
      <property name="text">
       <string>Открыть</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>290</y>
     <width>671</width>
     <height>33</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLineEdit" name="Zapros">
      <property name="tabletTracking">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="ZaprosButton">
      <property name="text">
       <string>Запрос  </string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>360</y>
     <width>671</width>
     <height>231</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="ConnectandInquiry">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>220</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:24pt;&quot;&gt;Подключение и Запросы&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="Database">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>160</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:24pt;&quot;&gt;База данных&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>190</y>
     <width>671</width>
     <height>33</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QLineEdit" name="CreateBDN">
      <property name="text">
       <string>Введите название базы данных:/Users/maksimgordienko/Qt1/Sql/RPM13/test</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="CreateBD">
      <property name="text">
       <string>Создать</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QCheckBox" name="NotNull">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>80</y>
     <width>86</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>НП</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="PrimaryKey">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>80</y>
     <width>86</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>ПК</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="AutoIn">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>80</y>
     <width>86</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>АИ</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="UNIQUE">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>80</y>
     <width>86</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>У</string>
   </property>
  </widget>
  <widget class="QPushButton" name="CreateColums">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>10</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Добавить столбец</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="tableName">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Название таблицы</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Update">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>10</y>
     <width>51</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>↺</string>
   </property>
  </widget>
  <widget class="QLabel" name="connectError">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>230</y>
     <width>391</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="nameColumn">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>80</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Название столбца</string>
   </property>
  </widget>
  <widget class="QComboBox" name="Type">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>80</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>INTEGER</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>TEXT</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>DATE</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="FillingOut">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:24pt;&quot;&gt;Заполнение&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>130</y>
     <width>671</width>
     <height>33</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLineEdit" name="tableName_2">
      <property name="text">
       <string>Название таблицы</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="nameColumn_2">
      <property name="text">
       <string>Название столбца</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="DataLine">
      <property name="text">
       <string>Данные</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="AddNote">
      <property name="text">
       <string>Добавить</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="DeleteNote">
      <property name="text">
       <string>Удалить</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="ErrorDate">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>110</y>
     <width>491</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="ErrorCreate">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>50</y>
     <width>541</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="DropColumn">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>10</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Удалить столбец</string>
   </property>
  </widget>
  <widget class="QLabel" name="ErrorDate_2">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>330</y>
     <width>491</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
